src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-advanced-dashboarding.mdx

```diff
@@ -0,0 +1,471 @@
+
+import queriesnrql2tutorial15 from 'images/queries-nrql_screenshot-crop-tutorial2-15.webp'
+
+import queriesnrql2tutorial16 from 'images/queries-nrql_screenshot-crop-tutorial2-16.webp'
+
+import queriesnrql2tutorial17 from 'images/queries-nrql_screenshot-crop-tutorial2-17.webp'
+
+import queriesnrql2tutorial18 from 'images/queries-nrql_screenshot-crop-tutorial2-18.webp'
+
+import queriesnrql2tutorial19 from 'images/queries-nrql_screenshot-crop-tutorial2-19.webp'
+
+import queriesnrql2tutorial20 from 'images/queries-nrql_screenshot-crop-tutorial2-20.webp'
+
+We hope seeing example queries and explanations in the context of your own data helps you better understand how to transform data into powerful visuals. In this tutorial, you'll use what you learned in our previous tutorials on the [foundations of using NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/) and [controlling your data with NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-control-your-data/) and build on them with more advanced concepts.
+
+In this tutorial, you'll learn how to advance your dashbaording by using faceting by case, advanced aggregation functions, the `EXTRAPOLATE` keyword, filtering aggregation functions, and overriding values. Specifically, you'll learn how to use:
+
+* Advanced aggregation functions like `filter()`, `apdex()`, `rate()funnel()`, `histogram()`.
+* The `EXTRAPOLATE` clauses.
+* `FACET CASES()`, including how to use attribute and group matching values.
+* `filter()` to combine Event Types.
+* Overriding values, when necessary.
+
+You may not use these functions and features on every single dashboard, but they will certainly come in handy when tackling specific problems and requirements. Let's get started!
+
+### Use advanced aggregators 
+<Steps>
+    <Step>
+#### Calculating Rate
+Let's start with the `rate()` function. It allows you to visualize the frequency of events over time, which helps when you want to understand the frequency of events in small periods of time within larger time windows.
+
+In the example below, you can see the average frequency of requests every 5 minutes for the last hour compared to the previous hour's 5-minute average frequency. Notice the query uses `SINCE 1 hour ago`, this is the overall time window in which you're calculating the rate.
+
+<SideBySide>
+    <Side>
+```sql
+SELECT rate(count(*), 5 minutes) 
+FROM Public_APICall 
+SINCE 1 hour ago 
+COMPARE WITH 1 hour ago
+```
+    </Side>
+    <Side>
+<img
+    title="Calculate range"
+    alt="A screenshot of a query using Since and Compare to capture data within a time range"
+    src={queriesnrql2tutorial1}
+/>
+    </Side>
+</SideBySide>
+
+You can use `rate()` to calculate requests per minute or requests per second by setting the time interval to either 1 minute or 1 second.
+
+    </Step>
+    <Step>
+#### Funnel Charts
```

Comment:
Can we give a bit more detail here? I'm sure these might be general data visualization concepts, but I'm wondering if we can zoom out. For example, instead of labeling something 'Timeseries', labelling the title 'See change over time with timeseries'.

These concepts seem pretty nested tho, but wanted to call. out that some customers might not grasp what 'funnel charts' is trying to convey and may skip past.